# Translations template for sahara.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the sahara project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2015.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: sahara 2015.1.dev216\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-03-17 06:46+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: sahara/config.py:178
#, python-format
msgid "Option '%(option)s' is required for config group '%(group)s'"
msgstr ""

#: sahara/config.py:186
msgid ""
"use_namespaces can not be set to \"True\" when use_neutron is set to "
"\"False\""
msgstr ""

#: sahara/context.py:145
msgid "Context isn't available here"
msgstr ""

#: sahara/exceptions.py:28
msgid "An unknown exception occurred"
msgstr ""

#: sahara/exceptions.py:38
#, python-format
msgid ""
"%(message)s\n"
"Error ID: %(id)s"
msgstr ""

#: sahara/exceptions.py:43
#, python-format
msgid "Object '%s' is not found"
msgstr ""

#: sahara/exceptions.py:58
msgid "Name already exists"
msgstr ""

#: sahara/exceptions.py:68
msgid "Invalid credentials"
msgstr ""

#: sahara/exceptions.py:78
msgid "Invalid object reference"
msgstr ""

#: sahara/exceptions.py:88
#, python-format
msgid "Error during command execution: \"%s\""
msgstr ""

#: sahara/exceptions.py:122
msgid "Data is invalid"
msgstr ""

#: sahara/exceptions.py:132
msgid "Job binary internal data must be a string of length greater than zero"
msgstr ""

#: sahara/exceptions.py:143
msgid ""
"To work with JobBinary located in internal swift add 'user' and "
"'password' to extra"
msgstr ""

#: sahara/exceptions.py:154
msgid "Database object already exists"
msgstr ""

#: sahara/exceptions.py:164
msgid "Object was not created"
msgstr ""

#: sahara/exceptions.py:174
msgid "Operation was not canceled"
msgstr ""

#: sahara/exceptions.py:184
msgid "Object was not deleted"
msgstr ""

#: sahara/exceptions.py:195
#, python-format
msgid "Node Group %s is missing 'floating_ip_pool' field"
msgstr ""

#: sahara/exceptions.py:215
#, python-format
msgid "Size of data (%(size)s) is greater than maximum (%(maximum)s)"
msgstr ""

#: sahara/exceptions.py:228
#, python-format
msgid "An error occurred in thread '%(thread)s': %(e)s"
msgstr ""

#: sahara/exceptions.py:239
#, python-format
msgid "Feature '%s' is not implemented"
msgstr ""

#: sahara/exceptions.py:246
#, python-format
msgid "Heat stack failed with status %s"
msgstr ""

#: sahara/exceptions.py:285
#, python-format
msgid "'%(operation)s' timed out after %(timeout)i second(s)"
msgstr ""

#: sahara/exceptions.py:289
#, python-format
msgid "Operation '%s'"
msgstr ""

#: sahara/exceptions.py:291
msgid "Operation"
msgstr ""

#: sahara/exceptions.py:308
msgid "You are not authorized to complete this action."
msgstr ""

#: sahara/exceptions.py:318
#, python-format
msgid "Image %s is not registered in Sahara"
msgstr ""

#: sahara/exceptions.py:327
#, python-format
msgid "Malformed message body: %(reason)s"
msgstr ""

#: sahara/exceptions.py:336
#, python-format
msgid ""
"Quota exceeded for %(resource)s: Requested %(requested)s, but available "
"%(available)s"
msgstr ""

#: sahara/exceptions.py:347
#, python-format
msgid "Object '%s' could not be updated"
msgstr ""

#: sahara/api/base.py:22
msgid "This API operation isn't implemented"
msgstr ""

#: sahara/api/middleware/auth_valid.py:54
msgid "Incorrect path"
msgstr ""

#: sahara/api/middleware/auth_valid.py:60
msgid "Token tenant != requested tenant"
msgstr ""

#: sahara/conductor/__init__.py:32
msgid "Remote conductor isn't implemented yet."
msgstr ""

#: sahara/conductor/resource.py:118
#, python-format
msgid "Unsupported type: %s"
msgstr ""

#: sahara/db/migration/cli.py:41
msgid "You must provide a revision or relative delta"
msgstr ""

#: sahara/db/sqlalchemy/api.py:271
#, python-format
msgid "Duplicate entry for Cluster: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:281
#, python-format
msgid "Duplicate entry for NodeGroup: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:293 sahara/db/sqlalchemy/api.py:305
#: sahara/db/sqlalchemy/api.py:324 sahara/db/sqlalchemy/api.py:1124
#, python-format
msgid "Cluster id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:340 sahara/db/sqlalchemy/api.py:352
#: sahara/db/sqlalchemy/api.py:371
#, python-format
msgid "Node Group id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:390 sahara/db/sqlalchemy/api.py:401
#: sahara/db/sqlalchemy/api.py:418 sahara/db/sqlalchemy/api.py:429
#, python-format
msgid "Instance id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:470
#, python-format
msgid "Duplicate entry for ClusterTemplate: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:481
#, python-format
msgid "Duplicate entry for TemplatesRelation: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:495 sahara/db/sqlalchemy/api.py:515
#, python-format
msgid "Cluster Template id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:498
#, python-format
msgid "Cluster template id '%s' is a default template"
msgstr ""

#: sahara/db/sqlalchemy/api.py:520
#, python-format
msgid "ClusterTemplate id '%s' can not be updated. It is a default template."
msgstr ""

#: sahara/db/sqlalchemy/api.py:532
#, python-format
msgid ""
"Cluster Template can not be updated. Another cluster template with name "
"%s already exists."
msgstr ""

#: sahara/db/sqlalchemy/api.py:540
#, python-format
msgid ""
"Cluster Template id '%s' can not be updated. It is referenced by at least"
" one cluster."
msgstr ""

#: sahara/db/sqlalchemy/api.py:591
#, python-format
msgid "Duplicate entry for NodeGroupTemplate: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:605
#, python-format
msgid "Node Group Template id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:608
#, python-format
msgid "Node group template id '%s' is a default template"
msgstr ""

#: sahara/db/sqlalchemy/api.py:622
#, python-format
msgid "NodeGroupTemplate id '%s' not found"
msgstr ""

#: sahara/db/sqlalchemy/api.py:626
#, python-format
msgid "NodeGroupTemplate id '%s' can not be updated. It is a default template."
msgstr ""

#: sahara/db/sqlalchemy/api.py:637
#, python-format
msgid ""
"Node Group Template can not be updated. Another node group template with "
"name %s already exists."
msgstr ""

#: sahara/db/sqlalchemy/api.py:649
#, python-format
msgid ""
"NodeGroupTemplate id '%s' can not be updated. It is referenced by an "
"existing cluster."
msgstr ""

#: sahara/db/sqlalchemy/api.py:722
#, python-format
msgid "Duplicate entry for DataSource: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:735
#, python-format
msgid "Data Source id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:739 sahara/db/sqlalchemy/api.py:939
msgid " on foreign key constraint"
msgstr ""

#: sahara/db/sqlalchemy/api.py:740
#, python-format
msgid "Data Source deletion failed%s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:829
#, python-format
msgid "Duplicate entry for JobExecution: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:841 sahara/db/sqlalchemy/api.py:853
#, python-format
msgid "JobExecution id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:910
#, python-format
msgid "Duplicate entry for Job: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:922 sahara/db/sqlalchemy/api.py:935
#, python-format
msgid "Job id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:940
#, python-format
msgid "Job deletion failed%s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:989
#, python-format
msgid "Duplicate entry for JobBinary: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:1010
#, python-format
msgid "JobBinary id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:1014
msgid "JobBinary is referenced and cannot be deleted"
msgstr ""

#: sahara/db/sqlalchemy/api.py:1062 sahara/db/sqlalchemy/api.py:1080
#, python-format
msgid ""
"Size of internal binary (%(size)sKB) is greater than the maximum "
"(%(maximum)sKB)"
msgstr ""

#: sahara/db/sqlalchemy/api.py:1092
#, python-format
msgid "Duplicate entry for JobBinaryInternal: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:1105
#, python-format
msgid "JobBinaryInternal id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:1145 sahara/db/sqlalchemy/api.py:1159
#: sahara/db/sqlalchemy/api.py:1174 sahara/db/sqlalchemy/api.py:1190
#: sahara/db/sqlalchemy/api.py:1205
#, python-format
msgid "Cluster Provision Step id '%s' not found!"
msgstr ""

#: sahara/openstack/common/periodic_task.py:46
#, python-format
msgid "Unexpected argument for periodic task creation: %(arg)s."
msgstr ""

#: sahara/openstack/common/policy.py:111
msgid "The JSON file that defines policies."
msgstr ""

#: sahara/openstack/common/policy.py:114
msgid "Default rule. Enforced when a requested rule is not found."
msgstr ""

#: sahara/openstack/common/policy.py:118
msgid ""
"Directories where policy configuration files are stored. They can be "
"relative to any directory in the search path defined by the config_dir "
"option, or absolute paths. The file defined by policy_file must exist for"
" these directories to be searched."
msgstr ""

#: sahara/openstack/common/policy.py:142
#, python-format
msgid "Policy doesn't allow %s to be performed."
msgstr ""

#: sahara/openstack/common/policy.py:239
#, python-format
msgid "Rules must be an instance of dict or Rules, got %s instead"
msgstr ""

#: sahara/openstack/common/sslutils.py:53
#, python-format
msgid "Unable to find cert_file : %s"
msgstr ""

#: sahara/openstack/common/sslutils.py:56
#, python-format
msgid "Unable to find ca_file : %s"
msgstr ""

#: sahara/openstack/common/sslutils.py:59
#, python-format
msgid "Unable to find key_file : %s"
msgstr ""

#: sahara/openstack/common/sslutils.py:62
msgid ""
"When running server in SSL mode, you must specify both a cert_file and "
"key_file option value in your configuration file"
msgstr ""

#: sahara/openstack/common/sslutils.py:101
#, python-format
msgid "Invalid SSL version : %s"
msgstr ""

#: sahara/plugins/base.py:103
#, python-format
msgid "Plugin with name '%s' already exists."
msgstr ""

#: sahara/plugins/base.py:114
#, python-format
msgid "Plugins couldn't be loaded: %s"
msgstr ""

#: sahara/plugins/exceptions.py:22
#, python-format
msgid "Chosen node group %(ng_name)s cannot be scaled : %(reason)s"
msgstr ""

#: sahara/plugins/exceptions.py:30
msgid "Failed to decommission cluster"
msgstr ""

#: sahara/plugins/exceptions.py:40
#, python-format
msgid "Cluster %(cluster_name)s cannot be scaled : %(reason)s"
msgstr ""

#: sahara/plugins/exceptions.py:51
#, python-format
msgid "Cluster is missing a service: %s"
msgstr ""

#: sahara/plugins/exceptions.py:54
#, python-format
msgid "%(message)s, required by service: %(required_by)s"
msgstr ""

#: sahara/plugins/exceptions.py:72
#, python-format
msgid ""
"Hadoop cluster should contain %(expected_count)s %(component)s "
"component(s). Actual %(component)s count is %(count)s"
msgstr ""

#: sahara/plugins/exceptions.py:94
#, python-format
msgid "Failed to Provision Hadoop Cluster: %s"
msgstr ""

#: sahara/plugins/exceptions.py:109
#, python-format
msgid "NameNode High Availability: %s"
msgstr ""

#: sahara/plugins/provisioning.py:118
#, python-format
msgid "Can't find applicable target '%(applicable_target)s' for '%(config_name)s'"
msgstr ""

#: sahara/plugins/provisioning.py:125
#, python-format
msgid "Can't find config '%(config_name)s' in '%(applicable_target)s'"
msgstr ""

#: sahara/plugins/utils.py:42 sahara/plugins/cdh/v5/validation.py:43
#: sahara/plugins/cdh/v5/validation.py:47
#: sahara/plugins/cdh/v5/validation.py:63
#: sahara/plugins/cdh/v5/validation.py:101
#: sahara/plugins/cdh/v5/validation.py:107
#: sahara/plugins/cdh/v5_3_0/validation.py:43
#: sahara/plugins/cdh/v5_3_0/validation.py:48
#: sahara/plugins/cdh/v5_3_0/validation.py:63
#: sahara/plugins/cdh/v5_3_0/validation.py:101
#: sahara/plugins/cdh/v5_3_0/validation.py:107
#: sahara/plugins/cdh/v5_3_0/validation.py:143
#: sahara/plugins/cdh/v5_3_0/validation.py:164
#: sahara/plugins/cdh/v5_3_0/validation.py:197
#: sahara/plugins/cdh/v5_3_0/validation.py:200
#: sahara/plugins/vanilla/hadoop2/validation.py:32
#: sahara/plugins/vanilla/hadoop2/validation.py:36
#: sahara/plugins/vanilla/hadoop2/validation.py:41
#: sahara/plugins/vanilla/hadoop2/validation.py:53
#: sahara/plugins/vanilla/hadoop2/validation.py:77
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:70
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:77
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:99
msgid "0 or 1"
msgstr ""

#: sahara/plugins/utils.py:73
msgid "Start the following process(es): {process}"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:81
msgid "Delete instances"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:93
msgid "Decommission nodes"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:101
msgid "Refresh DataNodes"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:106
msgid "Refresh YARNNodes"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:119
msgid "Deploy configs"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:128
msgid "Update configs"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:160
msgid "Create mgmt service"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:184
#: sahara/plugins/cdh/v5_3_0/cloudera_utils.py:81
msgid "'cluster' or 'instance' argument missed"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:204
#, python-format
msgid "Process %(process)s is not supported by CDH plugin"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:207
msgid "Await agents"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:229
msgid "Cloudera agents failed to connect to Cloudera Manager"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:235
#: sahara/plugins/vanilla/hadoop2/config.py:63 sahara/service/engine.py:138
msgid "Configure instances"
msgstr ""

#: sahara/plugins/cdh/commands.py:81
msgid "OS on image is not supported by CDH plugin"
msgstr ""

#: sahara/plugins/cdh/exceptions.py:26
#, python-format
msgid "CM API version not meet requirement: %s"
msgstr ""

#: sahara/plugins/cdh/exceptions.py:42
#, python-format
msgid "CM API error: %s"
msgstr ""

#: sahara/plugins/cdh/exceptions.py:57
#, python-format
msgid "CM API attribute error: %s"
msgstr ""

#: sahara/plugins/cdh/exceptions.py:72
#, python-format
msgid "CM API value error: %s"
msgstr ""

#: sahara/plugins/cdh/plugin.py:29
msgid ""
"The Cloudera Sahara plugin provides the ability to launch the Cloudera "
"distribution of Apache Hadoop (CDH) with Cloudera Manager management "
"console."
msgstr ""

#: sahara/plugins/cdh/plugin_utils.py:176
msgid "Install packages"
msgstr ""

#: sahara/plugins/cdh/plugin_utils.py:191
msgid "Start Cloudera Agents"
msgstr ""

#: sahara/plugins/cdh/plugin_utils.py:212
msgid "Configure Swift"
msgstr ""

#: sahara/plugins/cdh/plugin_utils.py:266
msgid "Start Cloudera manager"
msgstr ""

#: sahara/plugins/cdh/plugin_utils.py:285
#, python-format
msgid ""
"Cloudera Manager failed to start in %(timeout)s minutes on node "
"'%(node)s' of cluster '%(cluster)s'"
msgstr ""

#: sahara/plugins/cdh/plugin_utils.py:298
msgid "Configure OS"
msgstr ""

#: sahara/plugins/cdh/client/clusters.py:48
msgid "Either 'version' or 'fullVersion' must be specified"
msgstr ""

#: sahara/plugins/cdh/client/resource.py:84
#, python-format
msgid "Command %(method)s %(path)s failed: %(msg)s"
msgstr ""

#: sahara/plugins/cdh/client/resource.py:127
msgid "Get retry max time reached."
msgstr ""

#: sahara/plugins/cdh/client/types.py:124
#: sahara/plugins/cdh/client/types.py:348
#, python-format
msgid "API version %(minv)s is required but %(acv)s is in use."
msgstr ""

#: sahara/plugins/cdh/client/types.py:250
#, python-format
msgid "Invalid property %(attname)s for class %(classname)s."
msgstr ""

#: sahara/plugins/cdh/client/types.py:255
#, python-format
msgid "Attribute %(attname)s of class %(classname)s is read only."
msgstr ""

#: sahara/plugins/cdh/client/types.py:267
#, python-format
msgid ""
"Class %(class1)s does not derive from %(class2)s; cannot update "
"attributes."
msgstr ""

#: sahara/plugins/cdh/v5/cloudera_utils.py:93
#: sahara/plugins/cdh/v5_3_0/cloudera_utils.py:106
msgid "Create services"
msgstr ""

#: sahara/plugins/cdh/v5/cloudera_utils.py:121
#: sahara/plugins/cdh/v5_3_0/cloudera_utils.py:152
msgid "Configure services"
msgstr ""

#: sahara/plugins/cdh/v5/deploy.py:79 sahara/plugins/cdh/v5_3_0/deploy.py:82
msgid "Start roles: NODEMANAGER, DATANODE"
msgstr ""

#: sahara/plugins/cdh/v5/deploy.py:228
msgid "Create HBase common lib"
msgstr ""

#: sahara/plugins/cdh/v5/validation.py:130
#: sahara/plugins/cdh/v5/validation.py:133
#: sahara/plugins/cdh/v5_3_0/validation.py:130
#: sahara/plugins/cdh/v5_3_0/validation.py:133
msgid "at least 1"
msgstr ""

#: sahara/plugins/cdh/v5/validation.py:143
#: sahara/plugins/cdh/v5/validation.py:163
#: sahara/plugins/cdh/v5_3_0/validation.py:223
#: sahara/plugins/cdh/v5_3_0/validation.py:243
#, python-format
msgid "CDH plugin cannot scale nodegroup with processes: %(processes)s"
msgstr ""

#: sahara/plugins/cdh/v5/validation.py:149
#: sahara/plugins/cdh/v5_3_0/validation.py:229
msgid ""
"CDH plugin cannot scale node group with processes which have no master-"
"processes run in cluster"
msgstr ""

#: sahara/plugins/cdh/v5_3_0/cloudera_utils.py:100
msgid "First run cluster"
msgstr ""

#: sahara/plugins/cdh/v5_3_0/deploy.py:132
msgid "Prepare cluster"
msgstr ""

#: sahara/plugins/cdh/v5_3_0/deploy.py:145
msgid "Finish cluster starting"
msgstr ""

#: sahara/plugins/fake/plugin.py:30
msgid ""
"It's a fake plugin that aimed to work on the CirrOS images. It doesn't "
"install Hadoop. It's needed to be able to test provisioning part of "
"Sahara codebase itself."
msgstr ""

#: sahara/plugins/hdp/ambariplugin.py:231
msgid "An Ambari user in the admin group must be configured."
msgstr ""

#: sahara/plugins/hdp/ambariplugin.py:286
msgid ""
"The Hortonworks Sahara plugin automates the deployment of the Hortonworks"
" Data Platform (HDP) on OpenStack."
msgstr ""

#: sahara/plugins/hdp/clusterspec.py:325
#, python-format
msgid ""
"Could not determine property type for property '%(property)s' with value:"
" %(value)s"
msgstr ""

#: sahara/plugins/hdp/configprovider.py:65
#, python-format
msgid "Internal Error. Duplicate property name detected: %s"
msgstr ""

#: sahara/plugins/hdp/hadoopserver.py:88
msgid "Failed to install Hortonworks Ambari"
msgstr ""

#: sahara/plugins/hdp/hadoopserver.py:113
msgid "Failed to install Hadoop Swift integration"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/services.py:523
#: sahara/plugins/hdp/versions/version_2_0_6/services.py:602
#, python-format
msgid "Invalid value for property 'hbase-site/hbase.rootdir' : %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:95
#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:638
msgid ""
"The HDP plugin does not support the decommissioning of nodes for HDP "
"version 1.3.2"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:231
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:207
#, python-format
msgid "Failed to add cluster: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:280
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:256
#, python-format
msgid "Failed to set configurations on cluster: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:296
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:273
#, python-format
msgid "Failed to add services to cluster: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:314
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:292
#, python-format
msgid "Failed to add components to services: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:333
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:311
#, python-format
msgid "Failed to add host: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:349
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:330
#, python-format
msgid "Failed to add host component: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:375
#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:381
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:355
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:361
msgid "Installation of Hadoop stack failed."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:422
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:407
msgid "Unable to finalize Ambari state."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:453
#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:461
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:437
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:445
msgid "Start of Hadoop services failed."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:479
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:463
msgid "Failed to change state of Hadoop components"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:486
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:470
msgid "Hadoop/Ambari command failed."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:553
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:537
msgid ""
"Unable to determine installed service components in scaled instances.  "
"status code returned = {0}"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:558
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:542
msgid "Ambari agents registering with server"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:587
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:571
msgid "Unable to update Ambari admin user credentials: {0}"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:603
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:587
msgid "Unable to create Ambari user: {0}"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:613
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:597
#, python-format
msgid "Unable to delete Ambari user: %(user_name)s : %(text)s"
msgstr ""

#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:671
msgid ""
"An error occurred while trying to decommission the DataNode instances "
"that are being shut down. Please consult the Ambari server logs on the "
"master node for more information about the failure."
msgstr ""

#: sahara/plugins/mapr/plugin.py:23
msgid ""
"The MapR Distribution provides a full Hadoop stack that includes the MapR"
" File System (MapR-FS), MapReduce, a complete Hadoop ecosystem, and the "
"MapR Control System user interface"
msgstr ""

#: sahara/plugins/mapr/base/base_cluster_context.py:266
msgid "Service not found in services list"
msgstr ""

#: sahara/plugins/mapr/base/base_cluster_context.py:270
msgid "Can not map service"
msgstr ""

#: sahara/plugins/mapr/base/base_node_manager.py:114
#, python-format
msgid "Node failed to connect to CLDB: %s"
msgstr ""

#: sahara/plugins/mapr/base/base_node_manager.py:121
msgid "CLDB failed to start"
msgstr ""

#: sahara/plugins/mapr/domain/configuration_file.py:126
msgid "Template object must be defined"
msgstr ""

#: sahara/plugins/mapr/domain/service.py:158
msgid "Config missing 'name'"
msgstr ""

#: sahara/plugins/mapr/domain/service.py:162
#, python-format
msgid "Config '%s' missing 'value'"
msgstr ""

#: sahara/plugins/mapr/domain/service.py:188
msgid "Specify the version of the service"
msgstr ""

#: sahara/plugins/mapr/services/maprfs/maprfs.py:67
msgid "Specifies that MapR-DB is in use."
msgstr ""

#: sahara/plugins/mapr/util/validation_utils.py:25
#, python-format
msgid ""
"Hadoop cluster should contain at least %(expected_count)d %(component)s "
"component(s). Actual %(component)s count is %(actual_count)d"
msgstr ""

#: sahara/plugins/mapr/util/validation_utils.py:41
#, python-format
msgid ""
"Hadoop cluster should contain odd number of %(component)s but "
"%(actual_count)s found."
msgstr ""

#: sahara/plugins/mapr/util/validation_utils.py:51
#, python-format
msgid "Node \"%(ng_name)s\" is missing component %(component)s"
msgstr ""

#: sahara/plugins/mapr/util/validation_utils.py:52
#, python-format
msgid "%(message)s, required by %(required_by)s"
msgstr ""

#: sahara/plugins/mapr/util/validation_utils.py:67
#, python-format
msgid "%(service)s service cannot be installed alongside %(package)s package"
msgstr ""

#: sahara/plugins/spark/config_helper.py:251
#: sahara/plugins/storm/config_helper.py:37
#, python-format
msgid "Unable to get parameter '%(param_name)s' from service %(service)s"
msgstr ""

#: sahara/plugins/spark/edp_engine.py:25
#, python-format
msgid "Spark 1.0.0 or higher required to run spark %s jobs"
msgstr ""

#: sahara/plugins/spark/plugin.py:55
msgid ""
"This plugin provides an ability to launch Spark on Hadoop CDH cluster "
"without any management consoles."
msgstr ""

#: sahara/plugins/spark/plugin.py:76 sahara/plugins/spark/plugin.py:100
#: sahara/plugins/storm/plugin.py:75
msgid "1 or more"
msgstr ""

#: sahara/plugins/spark/plugin.py:83
#, python-format
msgid "%s or more"
msgstr ""

#: sahara/plugins/spark/plugin.py:84
#, python-format
msgid "Number of %(dn)s instances should not be less than %(replication)s"
msgstr ""

#: sahara/plugins/spark/plugin.py:235 sahara/plugins/storm/plugin.py:189
msgid "Push configs to nodes"
msgstr ""

#: sahara/plugins/spark/plugin.py:457 sahara/plugins/spark/plugin.py:471
#, python-format
msgid "Spark plugin cannot scale nodegroup with processes: %s"
msgstr ""

#: sahara/plugins/spark/plugin.py:481
#, python-format
msgid ""
"Spark plugin cannot shrink cluster because there would be not enough "
"nodes for HDFS replicas (replication factor is %s)"
msgstr ""

#: sahara/plugins/spark/scaling.py:31 sahara/plugins/spark/scaling.py:58
#: sahara/plugins/vanilla/hadoop2/scaling.py:140
#: sahara/plugins/vanilla/hadoop2/scaling.py:146
#: sahara/plugins/vanilla/v1_2_1/scaling.py:31
#: sahara/plugins/vanilla/v1_2_1/scaling.py:45
#, python-format
msgid "Decommission %s"
msgstr ""

#: sahara/plugins/spark/scaling.py:94
#: sahara/plugins/vanilla/hadoop2/scaling.py:134
#: sahara/plugins/vanilla/v1_2_1/scaling.py:82
#, python-format
msgid "Cannot finish decommission of cluster %(cluster)s in %(seconds)d seconds"
msgstr ""

#: sahara/plugins/storm/plugin.py:50
msgid ""
"This plugin provides an ability to launch Storm cluster without any "
"management consoles."
msgstr ""

#: sahara/plugins/vanilla/plugin.py:26
msgid ""
"The Apache Vanilla plugin provides the ability to launch upstream Vanilla"
" Apache Hadoop cluster without any management consoles. It can also "
"deploy the Oozie component."
msgstr ""

#: sahara/plugins/vanilla/hadoop2/config.py:353
msgid "Configure topology data"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/config_helper.py:161
#, python-format
msgid "Unable to get parameter '%(name)s' from service %(service)s"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/run_scripts.py:70
#, python-format
msgid "Process %s is not supported"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/run_scripts.py:121
#, python-format
msgid "Refresh %s nodes"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/run_scripts.py:163
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:209
#, python-format
msgid "Await %s start up"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/scaling.py:52
msgid "Update include files"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:71
msgid "Number of datanodes must be not less than dfs.replication."
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:88
#: sahara/plugins/vanilla/hadoop2/validation.py:107
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:514
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:533
#, python-format
msgid "Vanilla plugin cannot scale nodegroup with processes: %s"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:93
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:519
msgid ""
"Vanilla plugin cannot scale node group with processes which have no "
"master-processes run in cluster"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:116
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:543
#, python-format
msgid ""
"Vanilla plugin cannot shrink cluster because it would be not enough nodes"
" for replicas (replication factor is %s)"
msgstr ""

#: sahara/plugins/vanilla/v1_2_1/config_helper.py:212
#, python-format
msgid "Unable get parameter '%(parameter)s' from service %(service)s"
msgstr ""

#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:334
msgid "Setup instances and push configs"
msgstr ""

#: sahara/plugins/vanilla/v2_4_1/versionhandler.py:62
msgid ""
"The vanilla 2.4.1 plugin is now deprecated and will be removed in 2015.2 "
"release. The vanilla 2.6.0 plugin remains and  continues to be supported."
msgstr ""

#: sahara/service/direct_engine.py:166
msgid "Run instances"
msgstr ""

#: sahara/service/direct_engine.py:181
#, python-format
msgid "Server group with name %s is already exists"
msgstr ""

#: sahara/service/direct_engine.py:204
#, python-format
msgid "Several server groups with name %s found"
msgstr ""

#: sahara/service/direct_engine.py:291
msgid "Add instances"
msgstr ""

#: sahara/service/direct_engine.py:426
msgid "Wait for instances to become active"
msgstr ""

#: sahara/service/direct_engine.py:449
msgid "Wait for instances to be deleted"
msgstr ""

#: sahara/service/direct_engine.py:469
#, python-format
msgid "Node %s has error status"
msgstr ""

#: sahara/service/engine.py:72
msgid "Assign IPs"
msgstr ""

#: sahara/service/engine.py:94
msgid "Wait for instance accessibility"
msgstr ""

#: sahara/service/ops.py:238
msgid "Creating cluster failed for the following reason(s): {reason}"
msgstr ""

#: sahara/service/ops.py:272
msgid "Scaling cluster failed for the following reason(s): {reason}"
msgstr ""

#: sahara/service/ops.py:305
msgid "Terminating cluster failed for the following reason(s): {reason}"
msgstr ""

#: sahara/service/quotas.py:55
msgid "RAM"
msgstr ""

#: sahara/service/quotas.py:56
msgid "VCPU"
msgstr ""

#: sahara/service/quotas.py:57
msgid "instance"
msgstr ""

#: sahara/service/quotas.py:58
msgid "floating ip"
msgstr ""

#: sahara/service/quotas.py:59
msgid "security group"
msgstr ""

#: sahara/service/quotas.py:60
msgid "security group rule"
msgstr ""

#: sahara/service/quotas.py:61
msgid "port"
msgstr ""

#: sahara/service/quotas.py:62
msgid "volume"
msgstr ""

#: sahara/service/quotas.py:63
msgid "volume storage"
msgstr ""

#: sahara/service/trusts.py:67
msgid "Failed to create trust"
msgstr ""

#: sahara/service/trusts.py:107
msgid "Failed to delete trust {0}"
msgstr ""

#: sahara/service/validation.py:73
#, python-format
msgid "Object with %s not found"
msgstr ""

#: sahara/service/volumes.py:65
msgid "Attach volumes to instances"
msgstr ""

#: sahara/service/volumes.py:86
#, python-format
msgid "Error attach volume to instance %s"
msgstr ""

#: sahara/service/volumes.py:138
#, python-format
msgid "Volume %s has error status"
msgstr ""

#: sahara/service/volumes.py:167
msgid "Mount volumes to instances"
msgstr ""

#: sahara/service/edp/job_manager.py:96
#, python-format
msgid "Cluster does not support job type %s"
msgstr ""

#: sahara/service/edp/job_manager.py:181
#, python-format
msgid "Job execution %s was not canceled"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:51
#, python-format
msgid "Size of swift object (%(size)sKB) is greater than maximum (%(maximum)sKB)"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:68
#, python-format
msgid "Url for binary in internal swift must start with %s"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:74
msgid "Url for binary in internal swift must specify an object not a container"
msgstr ""

#: sahara/service/edp/oozie/workflow_creator/base_workflow.py:49
#, python-format
msgid "\"%s\" child cannot be added to prepare element"
msgstr ""

#: sahara/service/edp/oozie/workflow_creator/base_workflow.py:60
#, python-format
msgid "\"%s\" child cannot be added to streaming element"
msgstr ""

#: sahara/service/edp/spark/engine.py:314
#, python-format
msgid "Spark job execution failed. Exit status = %(status)s, stdout = %(stdout)s"
msgstr ""

#: sahara/service/heat/heat_engine.py:209
msgid "Create Heat stack"
msgstr ""

#: sahara/service/validations/base.py:68
#, python-format
msgid "Sahara doesn't contain plugin with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:74
#, python-format
msgid "Requested plugin '%(name)s' doesn't support version '%(version)s'"
msgstr ""

#: sahara/service/validations/base.py:81
#, python-format
msgid "Requested image '%s' is not registered"
msgstr ""

#: sahara/service/validations/base.py:92
#, python-format
msgid "Plugin doesn't contain applicable target '%s'"
msgstr ""

#: sahara/service/validations/base.py:97
#, python-format
msgid ""
"Plugin's applicable target '%(target)s' doesn't contain config with name "
"'%(name)s'"
msgstr ""

#: sahara/service/validations/base.py:145
msgid "Cinder is not supported"
msgstr ""

#: sahara/service/validations/base.py:168
#, python-format
msgid "Requested flavor '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:179
#, python-format
msgid "Security group '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:193
#, python-format
msgid "Floating IP pool %s not found"
msgstr ""

#: sahara/service/validations/base.py:197
#, python-format
msgid "Duplicates in node processes have been detected: %s"
msgstr ""

#: sahara/service/validations/base.py:213
#, python-format
msgid "Plugin doesn't support the following node processes: %s"
msgstr ""

#: sahara/service/validations/base.py:220
#, python-format
msgid "Duplicates in node group names are detected: %s"
msgstr ""

#: sahara/service/validations/base.py:228
#, python-format
msgid "Nova availability zone '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:236
#, python-format
msgid "Cinder availability zone '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:244
#, python-format
msgid "Volume type '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:252
#, python-format
msgid "Cluster with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:261
#, python-format
msgid "Cluster name '%s' is already used as Heat stack name"
msgstr ""

#: sahara/service/validations/base.py:273
#, python-format
msgid ""
"Composite hostname %(host)s in provisioned cluster exceeds maximum limit "
"%(limit)s characters"
msgstr ""

#: sahara/service/validations/base.py:284
#, python-format
msgid "Requested keypair '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:289
#, python-format
msgid "Network %s not found"
msgstr ""

#: sahara/service/validations/base.py:297
#, python-format
msgid "Cluster template with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:304
#, python-format
msgid "Cluster template with id '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:325
#, python-format
msgid "NodeGroup template with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:331
#, python-format
msgid "NodeGroup template with id '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:370
#, python-format
msgid "Cluster doesn't contain node group with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:384
#, python-format
msgid "Can't add new nodegroup. Cluster already has nodegroup with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:400
#, python-format
msgid ""
"Tags of requested image '%(image)s' don't contain required tags "
"['%(name)s', '%(version)s']"
msgstr ""

#: sahara/service/validations/cluster_templates.py:54
#, python-format
msgid "Cluster template %(id)s in use by %(clusters)s"
msgstr ""

#: sahara/service/validations/clusters.py:75
#: sahara/service/validations/clusters.py:99
#, python-format
msgid "'%s' field is not found"
msgstr ""

#: sahara/service/validations/clusters.py:93
msgid "'neutron_management_network' field can't be used with 'use_neutron=False'"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:80
#, python-format
msgid "Cluster created before Juno release can't be scaled with %(engine)s engine"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:87
#, python-format
msgid ""
"Cluster created with %(old_engine)s infrastructure engine can't be scaled"
" with %(new_engine)s engine"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:97
#, python-format
msgid "Requested plugin '%s' doesn't support cluster scaling feature"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:102
#, python-format
msgid "Cluster cannot be scaled not in 'Active' status. Cluster status: %s"
msgstr ""

#: sahara/service/validations/node_group_templates.py:50
#, python-format
msgid ""
"Node group template %(template)s is in use by cluster templates: "
"%(users)s; and clusters: %(clusters)s"
msgstr ""

#: sahara/service/validations/node_group_templates.py:60
msgid "You must specify a hadoop_version valuefor your plugin_name"
msgstr ""

#: sahara/service/validations/node_group_templates.py:64
msgid "You must specify a plugin_namefor your hadoop_version value"
msgstr ""

#: sahara/service/validations/plugins.py:27
#, python-format
msgid ""
"Requested plugin '%s' doesn't support converting config files to cluster "
"templates"
msgstr ""

#: sahara/service/validations/edp/base.py:53
#, python-format
msgid "Data source with name '%s' already exists"
msgstr ""

#: sahara/service/validations/edp/base.py:60
#, python-format
msgid "DataSource with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/edp/base.py:65
#, python-format
msgid "Job with name '%s' already exists"
msgstr ""

#: sahara/service/validations/edp/base.py:72
#, python-format
msgid "JobBinaryInternal with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/edp/base.py:80
#, python-format
msgid "Provided input and output DataSources reference the same location: %s"
msgstr ""

#: sahara/service/validations/edp/data_source.py:70
msgid "Swift url must not be empty"
msgstr ""

#: sahara/service/validations/edp/data_source.py:73
msgid "URL scheme must be 'swift'"
msgstr ""

#: sahara/service/validations/edp/data_source.py:82
#, python-format
msgid "URL must be of the form swift://container%s/object"
msgstr ""

#: sahara/service/validations/edp/data_source.py:86
msgid "No credentials provided for Swift"
msgstr ""

#: sahara/service/validations/edp/data_source.py:90
msgid "User is not provided in credentials for Swift"
msgstr ""

#: sahara/service/validations/edp/data_source.py:94
msgid "Password is not provided in credentials for Swift"
msgstr ""

#: sahara/service/validations/edp/data_source.py:99
msgid "HDFS url must not be empty"
msgstr ""

#: sahara/service/validations/edp/data_source.py:103
msgid "URL scheme must be 'hdfs'"
msgstr ""

#: sahara/service/validations/edp/data_source.py:106
msgid "HDFS url is incorrect, cannot determine a hostname"
msgstr ""

#: sahara/service/validations/edp/data_source.py:111
msgid "MapR FS url must not be empty"
msgstr ""

#: sahara/service/validations/edp/data_source.py:115
msgid "URL scheme must be 'maprfs'"
msgstr ""

#: sahara/service/validations/edp/job.py:69
#, python-format
msgid "Job binary '%s' does not exist"
msgstr ""

#: sahara/service/validations/edp/job.py:84
#, python-format
msgid "%s job requires main application jar"
msgstr ""

#: sahara/service/validations/edp/job.py:87
#, python-format
msgid "%s flow requires main script"
msgstr ""

#: sahara/service/validations/edp/job.py:91
msgid "'mains' and 'libs' overlap"
msgstr ""

#: sahara/service/validations/edp/job.py:96
#, python-format
msgid "%s flow requires libs"
msgstr ""

#: sahara/service/validations/edp/job.py:99
#, python-format
msgid "%s flow does not use mains"
msgstr ""

#: sahara/service/validations/edp/job_binary_internal.py:27
#, python-format
msgid "%s is not a valid name"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:58
#, python-format
msgid "%s job must specify edp.java.main_class"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:74
#, python-format
msgid "%s job must specify streaming mapper and reducer"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:83
#, python-format
msgid "Cluster with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:91
#, python-format
msgid "Cluster with id '%(cluster_id)s' doesn't support job type '%(job_type)s'"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:100
#, python-format
msgid "%s job requires 'input_id' and 'output_id'"
msgstr ""

#: sahara/topology/topology_helper.py:108
#, python-format
msgid "Unable to find file %s with compute topology"
msgstr ""

#: sahara/topology/topology_helper.py:124
#, python-format
msgid "Was not able to find compute node topology for VM %s"
msgstr ""

#: sahara/utils/api.py:140
msgid "Non-dict and non-empty kwargs passed to render"
msgstr ""

#: sahara/utils/api.py:162 sahara/utils/api.py:191
#, python-format
msgid "Content type '%s' isn't supported"
msgstr ""

#: sahara/utils/api.py:187
msgid "XML requests are not supported yet"
msgstr ""

#: sahara/utils/crypto.py:51
msgid "Private key file hasn't been created"
msgstr ""

#: sahara/utils/crypto.py:55
msgid "Public key file hasn't been created"
msgstr ""

#: sahara/utils/proxy.py:162
msgid "Proxy domain requested but not specified."
msgstr ""

#: sahara/utils/proxy.py:171
#, python-format
msgid "Failed to find domain %s"
msgstr ""

#: sahara/utils/proxy.py:175
#, python-format
msgid "Unexpected results found when searching for domain %s"
msgstr ""

#: sahara/utils/proxy.py:286
#, python-format
msgid "Failed to find user %s"
msgstr ""

#: sahara/utils/proxy.py:289
#, python-format
msgid "Unexpected results found when searching for user %s"
msgstr ""

#: sahara/utils/remote.py:140
msgid ""
"Remote driver is not loaded. Most probably you see this error because you"
" are running Sahara in distributed mode and it is broken.Try running "
"sahara-all instead."
msgstr ""

#: sahara/utils/ssh_remote.py:491
#, python-format
msgid "Unknown file mode %s"
msgstr ""

#: sahara/utils/ssh_remote.py:702
#, python-format
msgid "Closing HTTP session for %(host)s:%(port)s"
msgstr ""

#: sahara/utils/ssh_remote.py:708
#, python-format
msgid "Session for %(host)s:%(port)s not cached"
msgstr ""

#: sahara/utils/tempfiles.py:37
#, python-format
msgid "Failed to delete temp dir %(dir)s (reason: %(reason)s)"
msgstr ""

#: sahara/utils/types.py:88
#, python-format
msgid "Class %s is immutable!"
msgstr ""

#: sahara/utils/wsgi.py:253
msgid "cannot understand JSON"
msgstr ""

#: sahara/utils/wsgi.py:277
msgid "cannot understand XML"
msgstr ""

#: sahara/utils/openstack/base.py:45
#, python-format
msgid "Endpoint with type %(type)s is not found for service %(service)s"
msgstr ""

#: sahara/utils/openstack/base.py:52
#, python-format
msgid "Service \"%s\" not found in service catalog"
msgstr ""

#: sahara/utils/openstack/heat.py:56
#, python-format
msgid "Failed to find stack %(stack)s"
msgstr ""

#: sahara/utils/openstack/neutron.py:91
#, python-format
msgid "Neutron router corresponding to network %s is not found"
msgstr ""

